tchannel/tornado/broker.py

```diff
@@ -86,11 +89,13 @@ def send(self, client, endpoint, header, body, traceflag=None,
         except Exception as e:
             raise TChannelError(e.message)
 
+        protocol_headers = protocol_headers or {}
+        protocol_headers['as'] = self.arg_scheme.type()
```

Comment:
Is it possible that someone manually specifies the `as`? If yes, use `setdefault`? Otherwise this is fine.